test/plugins/features/movies/controller.test.js

```diff
@@ -23,5 +23,128 @@ describe('movie controller', () => {
     });
 
   });
-  
+
+
+  describe('retrieve a list of movies', () => {
+
+    it('returns all movies', () => {
+      return new Movie().fetchAll()
+      .then((movies) => {
+        return movies.length;
+      })
+      .then((count) => {
+        return [Controller.display(), count];
+      })
+      .spread((movies, count) => {
+        expect(movies).to.exist;
+        expect(movies).to.have.length(count);
+      });
+    });
+
+  });
+
+  describe('filters movies based on specified dates', () => {
+
+    it('returns movies in 2000', () => {
+      return new Movie().query(function(qb) {
+        qb.where('release_year', '=', '2000');
+      })
+      .fetchAll()
+      .then((movies) => {
+        return movies.length;
+      })
+      .then((count) => {
+        return [Controller.display({release_year: '2000'}), count];
+      })
+      .spread((movies, count) => {
+        expect(movies).to.exist;
+        expect(movies).to.have.length(count);
+
+        if (count !== 0) {
```

Comment:
This test shouldn't _need_ a guard since to properly test this filter behavior we need to assume that the response will be not empty. (This applies to the other guards in the file).